vendor/xclannad/wavfile.cc

```diff
@@ -738,7 +742,7 @@ void MP3FILE::Seek(int count) {
 	} while(pimpl->status != MP3FILE_impl::DONE && pimpl->write_pointer < pimpl->write_data_len);
 	return;
 }
-#elif USE_SMPEG
```

Comment:
Then MAD decoder likely wasn't tested as well. 77540b1 didn't check `HAVE_LIBMAD` is properly passed to`wavfile.cc`.